src/api/app/services/workflows/yaml_to_workflows_service.rb

```diff
@@ -21,8 +21,9 @@ def create_workflows
 
       workflows = []
 
-      parsed_workflows_yaml.each do |_workflow_name, workflow|
-        workflows << Workflow.new(workflow: workflow, scm_extractor_payload: @scm_extractor_payload, token: @token)
+      parsed_workflows_yaml.each do |_workflow_name, workflow_instructions|
```

Comment:
instead of iterator.each, saving the results in an array and returning the array, couldn't we use `map` or `select` ?

Reply:
Totally. On it.